### PR TITLE
Update @yamada-ui/breadcrumb accessibility

### LINKED ISSUE #1726: Update `@yamada-ui/breadcrumb` accessibility
### Description
This issue aims to address accessibility deficiencies in our `breadcrumb component` as per W3C guidelines. 
The component currently does not meet the accessibility standards outlined in the W3C documentation. 

The goal is to ensure the component is accessible to all users, including those using assistive technologies.
Ensure that all unchecked items in this checklist should be addressed.

#### General Requirements
- [x] **Navigation Landmark:**
  - [x] Breadcrumb trail is contained within a `navigation` landmark region.

- [ ] **Landmark Region Label:**
  - [ ] The navigation landmark region is labeled using `aria-label` or `aria-labelledby`.

- [x] **Current Page Indicator:**
  - [x] The link to the current page has `aria-current="page"`.

### Problem Statement/Justification

none

### Proposed Solution or API

_No response_

### Alternatives

_No response_

### Are you willing to participate in implementing this feature and create a pull request include the implementation?

None

### Additional Information

_No response_

### References

https://www.w3.org/WAI/ARIA/apg/patterns/breadcrumb/

### PR DESCRIPTION
**Pull request by cidkumagai**
Closes #1726  

## Description

Update @yamada-ui/breadcrumb accessibility

### CODE REVIEW

**Review by fffk10 — APPROVED** [review]
LGTM!

File: packages/components/breadcrumb/src/breadcrumb.tsx [hunk 1]
```diff
@@ -87,6 +87,7 @@ export const Breadcrumb = forwardRef<BreadcrumbProps, "nav">((props, ref) => {
   const [styles, mergedProps] = useMultiComponentStyle("Breadcrumb", props)
 
   const {
+    id: idProps,
```
[the remark this answers is not in the document]
Comment (setodeve) [reply packages/components/breadcrumb/src/breadcrumb.tsx:90]: @cidkumagai 

I'm sorry, maybe I misunderstanding.
but, Is it wrong to implement it as below?

`aria-label = "breadcrumb"`

[the remark this answers is not in the document]
Comment (cidkumagai) [reply packages/components/breadcrumb/src/breadcrumb.tsx:90]: If that implementation works, then I’d like to proceed with it.

[the remark this answers is not in the document]
Comment (cidkumagai) [reply packages/components/breadcrumb/src/breadcrumb.tsx]: @setodeve 
I have made the corrections based on your advice and would like to request a review.🙇‍♂️

**Review by setodeve — DISMISSED** [review]

File: packages/components/breadcrumb/src/breadcrumb.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (setodeve) [reply packages/components/breadcrumb/src/breadcrumb.tsx]: @cidkumagai 
I'sorry, Thanks!

**Review by fffk10 — DISMISSED** [review]

**Comment by hirotomoyamada** [comment]
`Aria-label` is a sentence. So the sentence starts with a capital letter.

**Comment by setodeve** [comment]
> @cidkumagai @fffk10 @setodeve
> 
> `Aria-label` is a sentence. So the sentence starts with a capital letter.


Do you mean `breadcrumb` should be changed to `Breadcrumb` as below?

`aria-label="Breadcrumb"`

**Comment by hirotomoyamada** [comment]
@setodeve @cidkumagai 

> Do you mean breadcrumb should be changed to Breadcrumb as below?

Yes.

**Comment by setodeve** [comment]
@cidkumagai 
sorry..
Could you change to `aria-label="Breadcrumb"`?

**Comment by cidkumagai** [comment]
@hirotomoyamada  @setodeve 
I have addressed the comments you provided and made the necessary corrections. Could you please review it again?

**Review by setodeve — APPROVED** [review]

**Review by fffk10 — APPROVED** [review]

**Pull request merged**